libs/arangodb/langchain_arangodb/chat_message_histories/arangodb.py

```diff
@@ -170,9 +199,23 @@ def add_message(self, message: BaseMessage) -> None:
                 "role": message.type,
                 "content": message.content,
                 "session_id": self._session_id,
+                "time": time.time(),
             },
         )
 
+    def add_qa_message(self, user_input: str, aql_query: str, result: str) -> None:
+        """Add a QA message to the chat history."""
```

Comment:
can we add the full docstring to this function, i.e `:param` and `:type` statements